go.mod

```diff
@@ -4,6 +4,7 @@ go 1.16
 
 require (
 	github.com/Jeffail/gabs v1.4.0
+	github.com/Jeffail/gabs/v2 v2.6.1
```

Comment:
are we using v1 and v2 in the codebase? maybe better to bump any other usage to v2 to avoid the extra dep and give us the update at the same time? 

We can also take care of it in another PR if you both agree and prefer.

Reply:
Yeah I agree it would be better to use v2 across all tests. I opted however for the pragmatic route and didn't touch tests that work just fine. I mostly wanted v2 for this for easier iteration over arrays.

My vote is to update to v2 if/when we touch the other tests.